nodeEditor/src/diagram/nodePort.ts

```diff
@@ -31,7 +32,11 @@ export class NodePort {
     }
 
     public get portName(){
-        return this.connectionPoint.displayName || this.connectionPoint.name;
+        let portName = this.connectionPoint.displayName || this.connectionPoint.name;
+        if (portName === "output") {

```

Comment:
same. We need a better way to detect that. Why not just checking if the ownerBlock.isInputBlock ?

Reply:
Updated